src/main/frontend/extensions/pdf/core.cljs

```diff
@@ -128,6 +128,7 @@
         left        (+ (:x point) (.-scrollLeft cnt))
         id          (:id highlight)
         new?        (nil? id)
+        can-open?   (or (not selection) (and selection (not= false (state/sub :pdf/auto-open-ctx-menu?))))
```

Comment:
Also curious if `(not= false ` is the convention of defensively checking a boolean ? (instead of simply `(and selection (state/sub :pdf/auto-open-ctx-menu?))`)

Reply:
If the `(not selection` is to handle existing selections, then this variable is checking for many things at the same time.

I would suggest checking the flag where it is used instead of creating this new var. Added a comment down below at the point of usage.